DayNightQ10.Rmd

```diff
@@ -50,7 +50,7 @@ print(final_results)
 ## calc_q10 Function
 ```{r Q10 Function}
 calc_q10 <- function(temp, resp){
-  positives <- resp >= 0
+  positives <- resp > 0 & !is.na(temp) & !is.na(resp)
   resp <- resp[positives]
   temp <- temp[positives]
   lnresp <- log(resp)
```

Comment:
Probably at this point should do `if(length(resp) < 4) return(NA)` or something...just in case we're left with no valid data.

Reply:
yes, I agree